rfcs/rfc-0002/README.md

```diff
@@ -0,0 +1,143 @@
+
+```
+# [PROPOSED NAME] Working Group Charter
+
+**Start Date**:
+**End Date**:
+**DRI**:
+
+## Motivation and Background
+Why is this working group necessary? What does it build upon? What does it enable? Is it related to other groups?
+
+## Scope
+What exactly is in scope and out of scope for the working group.
+
+## Deliverables
+Enumeration of work products for the group. Be as detailed as possible. 
+
+Make sure to classify the deliverables accordingly (e.g. specification, SDK, research report, etc.). Assign DRIs for each deliverable as necessary. Deliverables should each have clera acceptance criteria" so we know how to measure their completeness.
+
+### [Deliverable 1] – [Type]
+
+**Description**:
+
+**DRI**:
+
+**Scope of Work**:
+
+**Timeline/Milestones**:
+
+**Exit Criteria**:
+
+## Success Criteria
+How will we determine whether this group has achieved its objectives?
+
+## Coordination
+Will this group interact with other working groups? Other functions (Compliance, Legal, DevRel, OSPO, etc.). What resources will you require?
+
+## Communication
+How will this group organize itself and its work (which GitHub repos, projects, communication channels, etc.)? 
+
+How will progress be reviewed and reported to the TSC or other stakeholders? Include the frequency of reviews and the format of progress reports.
+```
+
+### Drawbacks
+
+**Overhead**: Working Groups / chartering could be seen as heavy handed. An alternative could be to have a set of work items approved by the TSC and not form working groups at all. This could lead to more overhead, slowing down decision-making and execution.
+
+Counterpoint: Having a distinct set of responsibilities and deliverables can be a powerful driver of focused delivery and accountability.
+
+**Flexibility**: Charters may limit the flexibility of Working Groups to pivot or adapt to new information or changing circumstances. A more flexible approach could allow for more dynamic and responsive project management.
+
+Counterpoint: Having a straightforward process for charter amendment could limit this risk.
+
+**Complexity in Coordination**: Managing multiple Working Groups with their own charters could create complexity in coordination, especially if there are interdependencies between groups. A more centralized approach might simplify coordination and reduce potential conflicts.
+
+Counterpoint: This trades off giving more autonomy and flexibility to those closest to the work.
+
+#### Alternative Approaches:
+
+**Ad Hoc Teams**: Instead of formal Working Groups, form ad hoc teams for specific tasks or projects. These teams can be more flexible and nimble, adapting quickly to changes without the need for formal charters.
+
+**Task Forces**: Establish task forces for urgent or high-priority initiatives. Task forces can have a narrow focus and short lifespan, providing a more agile approach to addressing specific issues.
```

Comment:
Appreciate the alternative approaches here. I'm not sure the distinction between ad hoc teams and task forces but FWIW it's an interesting consideration to begin with an ad hoc/task force team given the overhead of the Working Group. An ad hoc/task force team feels like the equivalent of an MVP compared with a Working Group - we may arrive to needing working groups ultimately, but task forces feel like less overhead and may be sufficient in achieving aims. Not sure, it's a good consideration.

Reply:
I thought of it like -- an ad-hoc team is spun up to _do the work_ (kind of like a scrum team, where a _task force_ would be more exploratory/short lived for a single (usually smaller in scope) piece of work

it is a bit abstract